src/utils/Cargo.toml

```diff
@@ -18,10 +18,10 @@ versionize = "0.1.10"
 versionize_derive = "0.1.6"
 vmm-sys-util = "0.11.2"
 vm-memory = { version = "0.13.0", features = ["backend-mmap", "backend-bitmap"] }
-log-instrument = { path = "../log-instrument", optional = true }
+log-instrument = { path = "../log-instrument" }
```

Comment:
can we have a regression test that verifies that?

Reply:
This would be this https://github.com/firecracker-microvm/firecracker/issues/4215 since it is a non-trivial task to write this test.

Reply:
makes sense. Would be worth adding a comment in the issue?